### DISCUSSION TITLE
Support inhibit_rules

### DISCUSSION THREAD

**Discussion by techwavedev** [grafana/grafana#37792] [Alerting]
I would like to have inhibit_rules as in Alertmanager 

Why is this needed:

The current way of cascading and nesting the notification groups is very tedious and confusing, I would like to be able to inhibit notifications based on inhibiting rules instead as we have an Alertmanager.

like:

inhibit_rules:

  - source_match:
      alertname: 'instance_down'

    target_match:
      alertname: 'ping_failed'

    equal: [ 'namespace', 'service' ]

**Comment by jadbaz** [comment]
Concerning this
> The current way of cascading and nesting the notification groups is very tedious and confusing, I would like to be able to inhibit notifications based on inhibiting rules

How can notification groups be used instead of inhibit_rules (even if not very practical)?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
